engine/common/rpc/convert/executor_metadata.go

```diff
@@ -0,0 +1,19 @@
+package convert
+
+import (
+	"github.com/onflow/flow/protobuf/go/flow/entities"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+func ExecutorMetadataToMessage(metadata flow.ExecutorMetadata) *entities.ExecutorMetadata {
```

Comment:
please add a `MessageToExecutorMetadata` as well, and a test that verifies converting back and forth returns the  same object.